### PR TITLE
[FQ2I] Add log op to FQ2I

### PR DESCRIPTION
**Pull request by CircleSpin**
Registered the log unary elementwise op

@mbrookhart @AndrewZhaoLuo @anwang2009

### CODE REVIEW

File: tests/python/relay/test_pass_fake_quantization_to_integer.py [hunk 1]
```diff
@@ -374,6 +377,19 @@ def test_fake_quantize_image_resize_bilinear():
...
+    x = relay.var("x", shape=[1, 3, 224, 224], dtype="int8")
+
+    zero = relay.const(0)
+    x = relay.qnn.op.dequantize(x, relay.const(2.0), zero)
+    op = relay.image.resize2d(x, size=[4, 4], method="linear")
+    op = relay.qnn.op.quantize(op, relay.const(2.0), zero)
+
+    x_np = np.random.randint(-128, 127, size=[1, 3, 224, 224], dtype="int8")
+
+    compare_fq_to_int(op, [x_np], allow_rounding_error=True)
```
Comment (anwang2009) [inline tests/python/relay/test_pass_fake_quantization_to_integer.py:390]: looks like this block shouldn't have been committed, remove?

**Review by AndrewZhaoLuo — COMMENTED** [review]
This will canonicalize to dq - log - q. 

Do you intend to add a more efficient implementation later?

File: python/tvm/relay/qnn/op/qnn.py [hunk 2]
```diff
@@ -998,6 +998,41 @@ def sigmoid(x, scale, zero_point, output_scale, output_zero_point):
     )
 
 
+def log(x, scale, zero_point, output_scale, output_zero_point):
+    """Quantized sigmoid.
```
[the remark this answers is not in the document]
Comment (CircleSpin) [reply python/tvm/relay/qnn/op/qnn.py:1002]: Thank you for catching that! :smiley:

File: tests/python/relay/test_pass_fake_quantization_to_integer.py (hunk 1 above)
Comment (CircleSpin) [reply tests/python/relay/test_pass_fake_quantization_to_integer.py:390]: Thank you! :)

File: python/tvm/relay/qnn/op/legalizations.py [hunk 3]
```diff
@@ -72,6 +72,7 @@ def legalize_qnn_unary_op(attrs, inputs, types):
 register_qnn_unary_op_legalize("qnn.erf", special.erf)
 register_qnn_unary_op_legalize("qnn.sigmoid", lambda arr: 1 / (1 + np.exp(-arr)))
 register_qnn_unary_op_legalize("qnn.tanh", np.tanh)
+register_qnn_unary_op_legalize("qnn.log", np.log)
```
Comment (mbrookhart) [inline python/tvm/relay/qnn/op/legalizations.py:75]: @AndrewZhaoLuo Doesn't this already do the lookup table?

**Review by mbrookhart — APPROVED** [review]

File: python/tvm/relay/qnn/op/legalizations.py (hunk 3 above)
Comment (AndrewZhaoLuo) [reply python/tvm/relay/qnn/op/legalizations.py:75]: oh yeah im dumb

**Pull request merged**